director/template/template.go

```diff
@@ -87,7 +87,7 @@ func (t Template) interpolateRoot(obj interface{}, tracker varsTracker) (interfa
 type interpolator struct{}
 
 var (
-	interpolationRegex         = regexp.MustCompile(`\(\((!?[-\.\w\pL]+)\)\)`)
+	interpolationRegex         = regexp.MustCompile(`\(\((!?[-\/\.\w\pL]+)\)\)`)
```

Comment:
Minor, but the forward slash doesn't need to be escaped. Maybe merger can change it if it isn't already changed.